test/README.md

```diff
@@ -175,6 +175,7 @@ Blueprint | Group | Image Name | Purpose
 rhel92.toml | group1 | rhel-9.2 | A simple RHEL image without MicroShift.
 rhel92-microshift-previous-minor.toml | group2 | rhel-9.2-microshift-4.13 | A RHEL 9.2 image with the latest MicroShift from the previous y-stream installed and enabled.
 rhel92-source.toml | group2 | rhel-9.2-microshift-source | A RHEL 9.2 image with the RPMs built from source.
+rhel92-source-base.toml | group2 | rhel-9.2-microshift-source-base | A RHEL 9.2 image with the RPMs built from base release branch.
```

Comment:
Should we use the same naming convention in the blueprint templates? el92_src-base.toml?

Reply:
That's a good idea, let's change it so it's easier to match visually